integration_test/images_test.go

```diff
@@ -186,7 +186,7 @@ var _ = BeforeSuite(func() {
 	imageDir, err = ioutil.TempDir("", "imagesTest")
 	Expect(err).To(BeNil())
 
-	imageStore, err = imagestore.NewImageStore(isoeditor.NewEditor(imageDir), imageDir, versions)
+	imageStore, err = imagestore.NewImageStore(isoeditor.NewEditor(imageDir), imageDir, true, versions)
```

Comment:
I'd expect this to false, right?
We would like to verify the official mirror I'd think.

Reply:
usually it's clearer to set a type for a boolean value, this way it's clear what is it used for, for example https://github.com/openshift/assisted-service/blob/80170758e2463f3b0f7e1c0256f1ebf7be57d0ab/internal/common/db.go#L119-L124

Reply:
@carbonin Yes! Thanks for catching that, I've changed it to `false` to verify the official mirror.

Thanks @filanov! I'll keep that in mind for future variables.